com.unity.cinemachine/Editor/Upgrader/UpgradeObjectToCm3.cs

```diff
@@ -86,8 +86,18 @@ public GameObject UpgradeComponents(GameObject go)
                      orbital.UpgradeToCm3(go.GetComponent<CinemachineOrbitalFollow>());
                      ConvertInputAxis(go, "Look Orbit X", ref orbital.m_XAxis, ref orbital.m_RecenterToTargetHeading);
                 }
+
                 if (ReplaceComponent<CinemachineTrackedDolly, CinemachineSplineDolly>(go))
-                    go.GetComponent<CinemachineTrackedDolly>().UpgradeToCm3(go.GetComponent<CinemachineSplineDolly>());
```

Comment:
Why was this not covered by the ordering of RootUpgradeComponentTypes?  Paths are supposed to get converted first.  What is the usecase that shows this problem?

I don't like this fix because it secretly upgrades 2 objects when this class is supposed to upgrade one object at a time.